aequilibrae/project/project.py

```diff
@@ -28,58 +32,97 @@ def __init__(self):
         self.conn: sqlite3.Connection = None
         self.network: Network = None
 
-    def load(self, file_name: str) -> None:
+    def open(self, project_path: str) -> None:
         """
         Loads project from disk
 
         Args:
-            *file_name* (:obj:`str`): Full path to the project data file. If does not exist, it will fail
+            *project_path* (:obj:`str`): Full path to the project data folder. If the project inside does
+            not exist, it will fail.
         """
-        self.path_to_file = file_name
+
+        if self.__other_project_still_open():
+            raise Exception('You already have a project open. Close that project before opening another one')
+
+        file_name = os.path.join(project_path, 'project_database.sqlite')
         if not os.path.isfile(file_name):
             raise FileNotFoundError("Model does not exist. Check your path and try again")
 
-        self.conn = sqlite3.connect(self.path_to_file)
+        self.project_base_path = project_path
+        self.path_to_file = file_name
         self.source = self.path_to_file
+        self.conn = sqlite3.connect(self.path_to_file)
         self.conn = spatialite_connection(self.conn)
-        self.network = Network(self)
+        self.__load_objects()
+        logger.info(f'Opened project on {self.project_base_path}')
 
-    def new(self, file_name: str) -> None:
+    def new(self, project_path: str) -> None:
         """Creates a new project
 
         Args:
-            *file_name* (:obj:`str`): Full path to the project data file. If file exists, it will fail
+            *project_path* (:obj:`str`): Full path to the project data folder. If folder exists, it will fail
         """
-        self.path_to_file = file_name
+        if self.__other_project_still_open():
+            raise Exception('You already have a project open. Close that project before creating a new one')
```

Comment:
oof, why are you using global singleton type patterns?
What resource are you protecting that only allows one instance of project to be created at once?

Reply:
This is motivated by being able to have the parameters file belonging to a project, instead of being global.

https://github.com/AequilibraE/aequilibrae/issues/116